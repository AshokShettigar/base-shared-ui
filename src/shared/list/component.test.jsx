import { shallow } from 'enzyme';

import List from './component';

const testName = 'test';
const defaultProps = {
    subList: jest.fn(),
    onSelect: jest.fn(),
    list: [
        {
            name: testName,
            icon: testName,
            id: 'id'
        }
    ],
    title: testName
};
const wrapper = shallow(
    <List
        {...defaultProps}
    />
);

it('should render component', () => {
    expect(wrapper).toMatchSnapshot();
});
