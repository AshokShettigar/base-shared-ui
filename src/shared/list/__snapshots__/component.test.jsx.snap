// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render component 1`] = `
<div
  className="root"
  data-component="List"
  data-id="list"
>
  <span
    className="title"
  >
    test
  </span>
  <ul
    className="list"
  >
    <li
      className="enabledItem"
      data-id="item"
      id="id"
      onClick={[Function]}
      role="presentation"
      tabIndex="0"
    >
      <WithRender
        className="bullet"
        glyph="test"
        glyphClassName="iconGlyph"
        hidden={false}
        rendered={true}
      />
      <span
        className="nameContainer"
      >
        <span
          className=""
          title="test"
        >
          test
        </span>
      </span>
    </li>
  </ul>
</div>
`;
